lib/ears/version.rb

```diff
@@ -1,3 +1,3 @@
 module Ears
-  VERSION = '0.8.2'
+  VERSION = '0.8.3'
```

Comment:
You should update the minor version here imo, as you add backwards compatible functionality. Patch versions are for bug-fixes or regular maintenance.

Reply:
Also hint: Afaik, our gem build pipeline only publishes new versions if the version update is the last commit of your branch when merged.

Reply:
Thanks for the hint 👍